kf_lib_data_ingest/common/constants.py

```diff
@@ -364,6 +365,10 @@ class CONSENT_TYPE:
     # General Research Use (NPU) (GRU-NPU) - Use of this data is
     # limited to general research use. Use of the data is limited to
     # not-for-profit organizations.
+    HMB_GSO = "HMB-GSO”
```

Comment:
Looks like the closing quote is breaking CI + linting. I think this should fix it
```suggestion
    HMB_GSO = "HMB-GSO"
```

Reply:
funky "smart quotes" might have gotten in there.  I thought that I had resolved it in the opening quote for the tests to pass, but seems that closing quote was still trying to be smart.  Thanks for the good eye, Chris!

Reply:
unsmarted the quote in https://github.com/kids-first/kf-lib-data-ingest/pull/658/commits/028e6836d6cddaa73bb8e8698ee0d0bda23bf538 ;-)